esinstall/src/index.ts

```diff
@@ -93,6 +93,12 @@ function resolveExportsMap(packageManifest: {exports: {}}, exportsProp: string)
     exportMapEntry?.require ||
     exportMapEntry;
 
+  // For now we are bailing on when this is an object, but in the future we should add support
+  // as Parcel seems to do something with an object here.
+  if (typeof exportMapValue === 'object') {
```

Comment:
Yea, this feels fine for now. We don't have an idea of development or production, so in effect this should be a "failed" lookup anyway.

https://nodejs.org/api/packages.html#packages_conditional_exports

To match Node.js behavior, we'll eventually want to go through these properties one by one looking for a match. So that `import: 'VALID'` would still be chosen even if `browser` were invalid. Can you either tackle that here, or add a TODO comment?

Reply:
re convo above: it sounds like Node.js behavior would be to not warn here: it's not an error, it's just an object with entrypoint keys ("development", "production") that our tool doesn't understand. So I'm fine matching that behavior here as well.

Reply:
Ah, good point about `import`, etc. possibly having a valid match, I'll redo to account for that.

Reply:
This spec is the living embodiment of why-not-both.gif